lambo

```diff
@@ -170,7 +177,9 @@ CODEEDITOR=""
 BROWSER=""
 TLD=$(php -r "echo json_decode(file_get_contents('$HOME/.valet/config.json'))->domain;")
 LINK=false
-
+CREATEDATABASE=false
+DBUSER=root
+BDPASSWORD=
```

Comment:
DB, not BD

Reply:
done! language derived typo DataBase vd. Base de Dados 😅